setup.py

```diff
@@ -53,10 +53,7 @@
         'mDNS',
     ],
     install_requires=[
-        # netifaces 0.10.5 has a bug that results in all interfaces' netmasks
-        # to be 255.255.255.255 on Windows which breaks things. See:
-        # * https://github.com/jstasiak/python-zeroconf/issues/84
-        # * https://bitbucket.org/al45tair/netifaces/issues/39/netmask-is-always-255255255255
-        'netifaces!=0.10.5',
+        'ifaddr',
+        'ipaddress'
```

Comment:
The ipaddress package dependency (both here and in requirements-dev.txt) needs to be removed as this project is Python 3.4+ only now and ipaddress is in the standard library since Python 3.3